index.html

```diff
@@ -199,15 +199,146 @@ <h1 class="header">What You Can Do at Technica</h1>
+                    experiences and challenges in tech. The goal of these meetups is to build community among underrepresented groups, 
+                    hear from diverse perspectives, and work to make tech a more inclusive space for all people. Technichats take place on 
+                    the main event-long livestream which can feature company representatives in a panel moderated by the Technica team. 
+                    This will include a live Q&A with our attendees and will be recorded and posted on the Technica youtube channel.
+                  </div>
+                </div>
+                <img class="content-img col-10 col-md-5 img-fluid" src="./img/technichats_cream.svg" />
+              </div>
+              <div class="row collapse-btn">View Less
+                <img class="arrow-img" src="./img/arrow_up.svg" />
+              </div>
           </div>
           <div id="network" class="tab-pane fade">
-            <p>Sed ut perspiciatis unde omnis iste natus error sit voluptatem accusantium doloremque laudantium, totam rem aperiam.</p>
+            <div class="content-wrapper row">
+              <div class="col-lg-6">
+                <div class="content-heading">Sponsorship Fair</div>
+                <div class="content-text">Looking to network? Companies are welcome to have recruiters and recruiting
+                   materials at their booth at Technica, where they will be able to interact with hackers. There will 
+                   be a Sponsorship Fair in which hackers are encouraged to visit open booths and network with recruiters, 
+                   potentially gaining an invitation to apply for openings or even an internship or job offer. 
+                </div>
+              </div>
+              <img class="content-img col-10 col-md-5 img-fluid" src="./img/sponsorfair_cream.svg" />
+            </div>
+            <div class="row expand-btn">Read More
+              <img class="arrow-img" src="./img/arrow_down.svg" />
+            </div>
+
+            <div class="content-wrapper row expanded-content">
+              <div class="col-lg-6">
+                <div class="content-heading">Chat With Mentors</div>
+                <div class="content-text">Need some advice? Mentors work closely with hackers throughout the weekend. 
+                  They assist with questions that arise, as well as problem-solve with hackers on their projects. 
+                  Mentors will connect with hackers across the country and make a significant impact in students’ introduction 
+                  into the world of technology. They are there to provide guidance and insights to participants, who choose to 
+                  create a software or hardware project during the hackathon.
+                </div>
+              </div>
+              <img class="content-img col-10 col-md-5 img-fluid" src="./img/flower-basket-icon.svg" />
```

Comment:
there's a chat with mentors image in [this folder](https://drive.google.com/drive/folders/1llHbNb9c0b9ONdn_iazxL2VY0Krea6FZ)